.build/tar/generate_tar.sh

```diff
@@ -16,7 +16,7 @@
 
 CONFIG_FILE=${CONFIG_FILE:-config-example.yaml}
 EXPORTERS_DIRECTORY=prometheus_exporters
-EXPORTERS_CONFIG_DIRECTORY=exporter_configs
```

Comment:
Also not directly related to the rest of this PR, I updated the name of this folder